guides/common/modules/con_managing-ansible-content.adoc

```diff
@@ -0,0 +1,6 @@
+[id="Managing_Ansible_Content{context}"]
+= Managing Ansible Content
+
+You can import Ansible collections from several sources to {ProjectServer}.
+
```

Comment:
```suggestion
```

IMHO a single sentence is not a "real" paragraph. Your call.

Reply:
I think I'd normally agree with you, but I'm modelling this after other introductions like this one: https://github.com/theforeman/foreman-documentation/blob/9169132a3ec8938066cef709eb135302cd91590c/guides/common/modules/con_managing-container-images.adoc?plain=1#L4